### ISSUE TITLE
Feature request: Ability to exclude a row from table-sort

### ISSUE THREAD

**Issue by leap0x7b** [LeeWannacott/table-sort-js#51]
I have icons in my table, which uses table-sort-js. I then noticed that you can sort by icon, which I didn't want. I read the documentation and there is no feature to exclude a row. That's why I requested this feature

**Comment by LeeWannacott** [comment]
Correct there is no feature for this.

You want to freeze certain table rows in place?, or just specific `<td>` in the row. Or disallow sorting on specific columns?
I should probably implement all three options. Do you have a screenshot you could share?

Thanks for opening the issue.

**Comment by nyenius** [comment]
Yes, me too I need the bottow row which contains 'Total' and blanks <td> to be excluded from the sort.

**Comment by LeeWannacott** [comment]
@nyenius Sorry for late reply :sweat_smile: I think the idiomatic way to do that would be to put the total column in a `<tfoot>` tag. Implement freezing row at some point : /
